pkg/frontend/frontend.go

```diff
@@ -67,15 +65,14 @@ type Runnable interface {
 }
 
 // NewFrontend returns a new runnable frontend
-func NewFrontend(ctx context.Context, baseLog *logrus.Entry, _env env.Interface, db *database.Database, apis map[string]*api.Version, m metrics.Interface, cipher encryption.Cipher, kubeActions kubeactions.Interface, resourcesClientFactory resourcesClientFactory) (Runnable, error) {
+func NewFrontend(ctx context.Context, baseLog *logrus.Entry, _env env.Interface, db *database.Database, apis map[string]*api.Version, m metrics.Interface, cipher encryption.Cipher, resourcesClientFactory resourcesClientFactory) (Runnable, error) {
```

Comment:
receive kubeActionsFamily here.

In non-test set kubeActionsFactory to kubeactions.New. In test we'll override it to some mock function.